test/kubemark/gce/util.sh

```diff
@@ -80,6 +81,30 @@ function create-kubemark-master {
             "${KUBE_ROOT}/hack/e2e-internal/e2e-grow-cluster.sh"
         done
     fi
+
+    # If cluster uses private master IP, two kubeconfigs are created:
+    # - kubeconfig with public IP, which will be used to connect to the cluster
+    #     from outside of the cluster network
+    # - kubeconfig with private IP (called internal kubeconfig), which will be
+    #      used to create hollow nodes.
+    #
+    # Note that hollow nodes might use either of these kubeconfigs, but
+    # using internal one is better from performance and cost perspective, since
+    # traffic does not need to go through Cloud NAT.
+    REGION=$(grep -o "^[a-z]*-[a-z0-9]*" "$KUBE_TEMP"/cluster-location.txt)
+    CLUSTER_NAME=$(cat "$KUBE_TEMP"/cluster-name.txt)
+    MASTER_NAME="$INSTANCE_PREFIX"-kubemark-master
```

Comment:
Same about this one?

Reply:
I don't see it set above, I've changed this var to use "KUBE_GCE_INSTANCE_PREFIX" which is set above explicitly (now we don't need to add the -kubemark part).